### ISSUE TITLE
Example should be renamed to include simpletest

### ISSUE THREAD

**Issue by kattni** [adafruit/Adafruit_CircuitPython_BusDevice#31]
Example `busdevice_read_register_i2c.py` could be renamed to `busdevice_read_register_i2c_simpletest.py` to match CircuitPython library standards.

The lines including this file name in the `docs/examples.rst` file must also be updated to reflect the name change, e.g.:
```
.. literalinclude:: ../examples/current_file_name.py
    :caption: examples/current_file_name.py
    :linenos:
```
would be updated to:
```
.. literalinclude:: ../examples/current_file_name_changed.py
    :caption: examples/current_file_name_changed.py
    :linenos:
```

**Comment by huyle101** [comment]
Hi @kattni 
I have already made PR for this issue. Could you please review it and let me know if there is any problem? Thank you.

**Comment by tannewt** [comment]
Fixed by #32. Thank you @huyle1097
